src/annotations/components/AnnotationCard.tsx

```diff
@@ -57,23 +47,27 @@ export const AnnotationCard: FC<Props> = ({annotationStream}) => {
     >
       <ResourceCard.Name
         onClick={handleNameClick}
-        name={annotationStream.name}
-        testID={`annotation-stream-card--name ${annotationStream.name}`}
+        name={annotationStream.stream}
+        testID={`annotation-stream-card--name ${annotationStream.stream}`}
+      />
+      <ResourceCard.Description
+        description={
+          annotationStream.description ??
+          'Add a description to your stream to see it here'
+        }
       />
-      <ResourceCard.Description description={annotationStream.description} />
       <ResourceCard.Meta>
         <>
           <Bullet
             glyph={IconFont.Annotate}
             size={ComponentSize.ExtraSmall}
             color={InfluxColors.Castle}
-            backgroundColor={annotationStream.display.color}
+            backgroundColor="#9078E4"
             style={{marginRight: '4px'}}
           />
-          Streaming from {annotationStream.query.bucketName}
         </>
-        <>Created at {annotationStream.meta.createdAt}</>
-        <>Last modified {annotationStream.meta.updatedAt}</>
+        <>Created at {annotationStream.createdAt ?? null}</>
+        <>Last modified {annotationStream?.updatedAt ?? null}</>
```

Comment:
>Isn't is safer to let it fallback?

It depends on what you mean by safer. read that pdf link if you haven't, it's a classic software engineering essay. It kind of contextualizes what safer means here. 

In this specific case, there's two things. One, if you fallback from `undefined` to `null`, is that really a fallback? A fallback is usually a helpful, actionable message to the user. Showing `null` isn't really that.

Higher level, what would a good fallback here be? Let's say if we didn't get the dates when displaying these streams. What should we show? a `-`? What does that represent? That there are no start and end times set? That they weren't set? That we didn't fetch them properly? A message? "Sorry, we couldn't fetch your dates"? Nobody cares if we can't fetch our dates.

If we can't get meta dates about an annotation when we fetch and display them, we have bigger problems that will likely be very obvious.

Tell me, under what circumstances would the dates not be defined, but everything else is?

Reply:
Created at an updated at are currently not required values in the annotations streams objects. So it is possible they aren't there. 

And currently, it will just say created at <blank>

Reply:
>Created at and updated at are currently not required values in the annotations streams objects. So it is possible they aren't there.

when is it possible? give me some examples. tell me the circumstances when they wouldn't be sent across the wire, or when they wouldn't be defined on the object? have you run into this locally in your testing? i haven't seen a 200 annotation request without dates on the annotation, but i haven't looked hard.